packages/atomic/.storybook/default-result-component-story.tsx

```diff
@@ -130,19 +136,82 @@ export default function defaultResultComponentStory(
 
   const defaultLoader = initializeInterfaceDebounced(() => {
     return `${renderArgsToResultTemplate(
-      renderArgsToHTMLString(componentTag, getArgs(), advancedConfig) +
+      renderArgsToHTMLString(componentTag, getArgs(), config) +
         renderShadowPartsToStyleString(componentTag, getArgs()),
       getArgs,
       true
-    )}${
-      advancedConfig.additionalMarkup
-        ? advancedConfig.additionalMarkup().strings.join('')
-        : ''
-    }`;
-  }, advancedConfig.engineConfig);
+    )}${renderAdditionalMarkup(config.additionalMarkup)}`;
+  }, config.engineConfig);
 
   exportedStory.loaders = [defaultLoader];
   exportedStory.decorators = [defaultDecorator];
 
   return {defaultModuleExport, exportedStory};
 }
+
+const buildConfigWithDefaultValues = (
+  advancedConfig: DefaultStoryAdvancedConfig
+) => {
+  return {
+    ...advancedConfig,
+    additionalMarkup: buildConfigAdditionalMarkup(advancedConfig),
+    engineConfig: {
+      ...advancedConfig.engineConfig,
+      preprocessRequest: buildConfigPreprocessRequest(advancedConfig),
+    },
+  };
+};
+
+const buildConfigAdditionalMarkup = (
+  advancedConfig: DefaultStoryAdvancedConfig
+) => {
+  const templateForMaxWidth = html`
+    <style>
+      atomic-search-interface,
+      atomic-result-list {
+        max-width: 1024px;
+        display: block;
+        margin: auto;
+      }
+    </style>
+  `;
+
+  if (advancedConfig.additionalMarkup) {
+    return () =>
+      html`${templateForMaxWidth}${advancedConfig.additionalMarkup()}`;
+  }
+  return () => templateForMaxWidth;
+};
+
+const buildConfigPreprocessRequest = (
+  advancedConfig: DefaultStoryAdvancedConfig
+) => {
+  const preprocessRequestForOneResult = (r) => {
+    const bodyParsed = JSON.parse(r.body as string);
+    bodyParsed.numberOfResults = 1;
```

Comment:
Could a single result have been set using the ResultsPerPage controller?

Reply:
Yes, however there's no easy way with Atomic (today) to get access to Headless functionalities.
So it would actually be more complicated.

Jean Felix has logged a story regarding this type of problem for Atomic IIRC.

( https://coveord.atlassian.net/browse/KIT-1164 )